src/nvim/eval/funcs.c

```diff
@@ -7343,7 +7343,7 @@ static void f_settagstack(typval_T *argvars, typval_T *rettv, EvalFuncData fptr)
   // third argument: action - 'a' for append and 'r' for replace.
   // default is to replace the stack.
   if (argvars[2].v_type == VAR_UNKNOWN) {
-    action = 'r';
+    // action = 'r';
```

Comment:
You should remove the line instead of just commenting it out.

Reply:
Why?

Reply:
Lol, I guess not